### ISSUE TITLE
"Fjern filter" used in English

### ISSUE THREAD

**Issue by Caroline-A** [SDGforskning/find-sdg-research#115]
I noticed today that the "clear filters" button is in Norwegian when the website is in English (I'm guessing at some point in the accessibility statement evaluation it was pointed out that it should be in Norwegian, and it got changed for both languages...?)

![image](https://github.com/SDGforskning/find-sdg-research/assets/42732381/e8283fe2-7002-41f9-bc0d-a17a255b0b1d)

**Comment by tarjelavik** [comment]
Failed to deconstruct the `useRouter()` object :-(
